class-metamodel/class-and-attribute-subsystem/class-attribute.xmm

```diff
@@ -12,8 +12,8 @@ subsystem Class And Attribute
 class Alias
 attributes
     Name : Alias Name {I}
-    Class {I, R29}
-    Domain {I, R29}
+    Class {I2, R29}
+    Domain {I, I2, R29}
```

Comment:
This missing ‘I2’ identifier is not real error (I could not help myself)

Reply:
It actually is a real error, thanks Erik!  (All logical differences are big differences – Ludwig Wittgenstein)